code/modules/mob/living/death.dm

```diff
@@ -88,6 +88,9 @@
 	if(mind)
 		mind.store_memory("Time of death: [station_time_timestamp("hh:mm:ss", timeofdeath)]", 0)
 		GLOB.respawnable_list += src
+		for(var/obj/item/I in mind.summoned_items)
+			I.loc.visible_message("<span class='warning'>[I] suddenly disappears!</span>")
```

Comment:
if you're going to have the item have the `visible_message`, do that, otherwise, you should be using the turf.